Examples/Swift/ViewController.swift

```diff
@@ -154,8 +154,9 @@ class ViewController: UIViewController, MGLMapViewDelegate, CLLocationManagerDel
             multipleStopsAction.isEnabled = false
         }
 
+        let destinationName = "Dropped Pin" // Note you can change this. This will be used in the top banner
         let coordinates = mapView.convert(tap.location(in: mapView), toCoordinateFrom: mapView)
-        let waypoint = Waypoint(coordinate: coordinates)
+        let waypoint = Waypoint(coordinate: coordinates, name: destinationName)
```

Comment:
Want to use `waypoints.count` here to show this can change for a multi-waypoint route?